exporter/views.py

```diff
@@ -84,74 +154,65 @@ def get(self, request, *args, **kwargs):
         )
 
 
-# FIXME: we should be able to export at the project and item level, too
-class ExportCampaignToBagit(TemplateView):
-    """
-    Creates temp directory structure for source data.
-    Executes bagit.py to turn temp directory into bagit strucutre.
-    Builds and exports bagit structure as zip.
-    Uploads zip to S3 if configured.
-    Removes all temporary directories and files.
-    """
+class ExportItemToBagIt(TemplateView):
+    @method_decorator(staff_member_required)
+    def get(self, request, *args, **kwargs):
+        campaign_slug = self.kwargs["campaign_slug"]
+        project_slug = self.kwargs["project_slug"]
+        item_id = self.kwargs["item_id"]
+
+        asset_qs = Asset.objects.filter(
+            item__project__campaign__slug=campaign_slug,
+            item__project__slug=project_slug,
+            item__item_id=item_id,
+            transcription_status="completed",
```

Comment:
This should probably use the same constants we're trying to use elsewhere:

```suggestion
            transcription_status=TranscriptionStatus.COMPLETED,
```